docs/usage/examples/opentelemetry.md

```diff
@@ -130,7 +130,7 @@ You should be able to see now the full trace view which shows each HTTP request
 ### Metrics
 
 Additional to the received traces some metrics are calculated.
-This is achieved using the [spanmetricsprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor).
+This is achieved using the [spanmetricsprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/3cd5b15466fe4065ef76ea5c557dab435a7d7318/processor/spanmetricsprocessor).
```

Comment:
do we really want to pin this?

Reply:
The current link is broken, `spanmetricprocessor` was deprecated and removed